scriptworker/test/data/cotv2/private_github_.taskcluster.yml

```diff
@@ -0,0 +1,294 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file should probably go into `cotv4`, which is the latest data. We might drop the `cotv2` test data directory when we drop non-action-hook support.